packages/devtools/src/microsoft_teams/devtools/web/index.html

```diff
@@ -5,8 +5,8 @@
     <link rel="icon" type="image/svg+xml" href="/devtools/icon.png" />
```

Comment:
Is there a file that needs to be removed?

Reply:
I thought I removed  it but not sure why the diff isn't capturing it. let me db check